infrastructure/variables.tf

```diff
@@ -69,6 +69,10 @@ variable scheduling_enabled {
   default = "false"
 }
 
+variable "file_cleanup_enabled" {
+  default = "false"
```

Comment:
would be nice to have short description :)

Reply:
done